src/hotspot/share/gc/z/zMappedCache.cpp

```diff
@@ -118,20 +118,20 @@ static ZMappedCacheEntry* create_entry(const ZVirtualMemory& vmem) {
   return entry;
 }
 
-bool ZMappedCache::EntryCompare::cmp(const IntrusiveRBNode* a, const IntrusiveRBNode* b) {
+bool ZMappedCache::EntryCompare::less(const IntrusiveRBNode* a, const IntrusiveRBNode* b) {
```

Comment:
I feel `less_than` sounds more natural as a method name; it also avoids "conflicting" with `RBTreeOrdering::less`.